datadog/dogshell/__init__.py

```diff
@@ -71,6 +71,14 @@ def main():
         "-v", "--version", help="Dog API version", action="version", version="%(prog)s {0}".format(__version__)
     )
 
+    parser.add_argument(
+        "--site",
+        help="Datadog site to send data, us (datadoghq.com), eu (datadoghq.eu) or us3 (us3.datadoghq.com), default: us",
+        dest="site",
+        default=os.environ.get("DD_SITE", os.environ.get("DATADOG_HOST")),
```

Comment:
Site and host have different semantic, I don't think we should use the same env var for this.